app/views/content_items/html_publication/_print_meta_data.html.erb

```diff
@@ -13,21 +13,9 @@
 <p>
   This publication is available at <%= @content_item.full_path(request) %>
 </p>
-<% unless @content_item.print_meta_data_contact_address.blank? %>
-    <p>
-      Any enquiries regarding this publication should be sent to us at:<br/>
-    </p>
-    <%= simple_format(@content_item.print_meta_data_contact_address) %>
-<% end %>
 
 <% unless @content_item.isbn.blank? %>
     <p>
       Print ISBN: <%= @content_item.isbn %>
```

Comment:
It would make sense to remove `Print` here, although that would be a user-facing change. Perhaps raise in a separate PR, as it's confusing to see it used without context of knowing there used to be another ISBN.

Reply:
Yeah, good point, I'll raise a separate one after this :)